lib/iex/lib/iex/dochelp/erlang_stub.ex

```diff
@@ -0,0 +1,48 @@
+defmodule IEx.DocHelp.ErlangStub do
+  @moduledoc """
+  Stub module to suggest possible addons for Erlang documentation.
+
+  Currently only returns {:unknown, doc_list }
+  with a suggestive error message.
+  """
+
+  @suggestive "and currently there is no helper installed to provide Erlang documentation"
```

Comment:
My intent was to only duplicate existing h behavoiur in this PR and then to submit a PR about possible replacements for ErlangStub. See https://github.com/bbense/doc_helpers/blob/master/lib/print_url.ex

for something that might replace ErlangStub.